src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -45,6 +45,32 @@ public static Index parseIndex(String oneBasedIndex) throws IllegalValueExceptio
     }
 
     /**
+     * Parses {@code oneBasedIndex} into an {@code Index[]} and returns it. Leading and trailing whitespaces will be
+     * trimmed.
+     * @throws IllegalValueException if the specified index is invalid (not non-zero unsigned integer).
+     */
+    public static Index[] parseDeleteIndex(String oneBasedIndex) throws IllegalValueException {
+        String[] parts = oneBasedIndex.split(",");
+        String[] trimmedIndex = new String[parts.length];
+        int[] trimmedIntIndex = new int[parts.length];
+        int i = 0;
+        for(String str : parts) {
```

Comment:
Use for(int i; i < parts.length; i++) so you can use parts[i] and trimmedIndex[i] instead of declaring 'i' outside the loop